README.md

```diff
@@ -71,7 +75,6 @@ The following example is tested on a minikube, so that you can play with this op
 
 As a result, a config map wrapped by `deployable`,  at default namespace. At the meantime, it will also deploy a `channel` resource at the `ch-ns` namespace.
```

Comment:
First part of line 76 not a complete sentence and I was not sure how to rewrite, maybe:

As a result, a config map is wrapped by a `deployable` at `default` namespace.  (?)

Not sure. Please commit as you see fit to fix that line.

Reply:
`As a result, a config map is wrapped by a deployable at default namespace.`

Will do.